sourmash/signature.py

```diff
@@ -83,6 +83,7 @@ def _display_name(self, max_length):
 
     def _save(self):
         "Return metadata and a dictionary containing the sketch info."
+        notify("in signature._save", end="\r")
```

Comment:
Can this notification be removed, or updated to be less developer-focused? You can use `debug` instead of `notify`, too.

Reply:
(`notify` is meant to be human-readable stuff that goes to stderr.)